skbeam/core/roi.py

```diff
@@ -691,3 +693,54 @@ def lines(end_points, shape):
         label += 1
         label_array[rr, cc] = label
     return label_array
+
+
+def auto_find_center_rings(avg_img, sigma=1, no_rings=4, min_samples=3,
+                           residual_threshold=1):
+    """This will find the center of the speckle pattern and the radii of the
+    most intense rings
```

Comment:
Please add more doc like this is based on ransac algorithm. Are you going to add other algorithms? Or ransac is the most standard in the field?